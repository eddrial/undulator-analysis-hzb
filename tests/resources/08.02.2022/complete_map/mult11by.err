 error report for the data set mult11by.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1590 data points 
        out of totally   2558 data points
 the rms value of the residuals is  0.44522E-02
 the maximum residual is  0.18491E-01
 the rms value of the lambda/2-filtered data is  0.36072E-01
 the maximum value of the lambda/2-filtered data is  0.27924E+00
