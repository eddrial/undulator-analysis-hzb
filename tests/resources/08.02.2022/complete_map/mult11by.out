mult11by.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1  1.0155857E+00
   2 -1.0313341E-02
   3 -9.7303698E-03
   4 -1.4219171E-04
   5 -2.5964642E-04
   6  5.5981346E+01
   7  1.0438947E+03
   8  1.0436332E+03
   9  1.0439869E+03
  10  3.6201304E-01
  11 -4.1282449E+00
 number of iterations:   13
     sumsq:  3.15163E-02
       rms:  4.45214E-03  dimension of magnetic field
       rms:  4.38378E-01  percent of b_eff
   rms_ext:  3.70901E-01  percent of b_eff
 B_eff =   1.0156E+00
 B_max =   9.9600E-01
 K_eff =    5.312
 lamda_1 (nm) =     38.37974

        eigenvectors in parameter space

 error                         direction
     1.42,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     0.60,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     0.05,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.03,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.00,  0.68 -0.44 -0.49 -0.19 -0.22 -0.07  0.00  0.00  0.00  0.00  0.00
     0.00,  0.70  0.63  0.28  0.14  0.13  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.04  0.04 -0.53  0.80  0.27  0.02  0.00  0.00  0.00  0.00  0.00
     0.00,  0.14 -0.41  0.23 -0.11  0.86 -0.04  0.00  0.00  0.00  0.00  0.00
     0.00,  0.15 -0.48  0.58  0.54 -0.34 -0.04  0.00  0.00  0.00  0.00  0.00
     0.00,  0.06 -0.06  0.01 -0.01  0.00  1.00  0.05  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:   0.00521 Tmm
 kick_2:  -0.00521 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.30675E+02
      
 the truncated data set has only   1590 data points 
      
 the rms value of the residuals is  0.44522E-02
 the maximum residual is  0.18491E-01
      
 the rms value of the lambda/2-filtered data is  0.36072E-01
 the maximum value of the lambda/2-filtered data is  0.27924E+00
      
 the rms phase error (all data) is    3.512 deg.
 the maximum phase error (all data) is    8.866 deg.
      
 the rms phase error (extrema) is    4.464 deg.
 the maximum phase error (extrema) is    9.535 deg.
      
 the Walker-rms phase error (all data) is   19.345 deg.
 the Walker-maximum phase error (all data) is   34.632 deg.
      
 the Walker-rms phase error (extrema) is    3.616 deg.
 the Walker-maximum phase error (extrema) is    7.731 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             3.511531    
 local phase error             2.216508    
 slowly varying phase error    2.658625    
 oscillatory phase error      0.5913147    
 -----------------------------------------
